lib/src/main/java/com/orange/ods/compose/component/textfield/password/OdsPasswordTextField.kt

```diff
@@ -88,7 +88,10 @@ fun OdsPasswordTextField(
     characterCounter: (@Composable () -> Unit)? = null
 ) {
     val odsPasswordTextFieldState = rememberOdsPasswordTextFieldState()
-
+    if (!visualisationIcon) {
+        odsPasswordTextFieldState.passwordVisible.value = false
+    }
```

Comment:
We can factorize this behavior by passing the boolean parameter `visualisationIcon` to the `rememberOdsPasswordTextFieldState()`
```kotlin
    val odsPasswordTextFieldState = rememberOdsPasswordTextFieldState(visualizationIcon = visualisationIcon)
```
 and in `OdsPasswordTextFieldState.kt`
 ```kotlin
    val isPasswordVisible
        get() = if (!visualizationIcon) false else passwordVisible.value
```